### ISSUE TITLE
Cleaning up cocopp.<TAB>

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#1578]
The title says kind of everything, see also the comment of @nikohansen in #1353.

Currently, `cocopp.<TAB>` gives
```
cocopp.bestalg       cocopp.config              cocopp.info
cocopp.captions      cocopp.dataformatsettings  cocopp.load
cocopp.cococommands  cocopp.findfiles           cocopp.main
cocopp.comp2         cocopp.genericsettings     cocopp.pickle
cocopp.compall       cocopp.htmldesc            cocopp.ppconverrorbars
```
for me.

From my point of view, `bestalg`, `config`, `genericsettings`, `load`, and `main` have to stay for sure (while `main` should be clearly renamed in the long run). `pickle` and `htmldesc` must go out. For the others, we shall discuss (but I am fine if they go out as well).

That gives, for the moment, the following list of things that shall stay:
  - [x] `cocopp.bestalg`
  - [x] `cocopp.config`
  - [ ] `cocopp.info`
  - [ ] `cocopp.captions`
  - [ ] `cocopp.dataformatsettings`
  - [x] `cocopp.load`
  - [ ] `cocopp.cococommands`
  - [ ] `cocopp.findfiles`
  - [x] `cocopp.main` (to be renamed)
  - [ ] `cocopp.comp2`
  - [x] `cocopp.genericsettings`
  - [ ] `cocopp.pickle`
  - [ ] `cocopp.compall`
  - [ ] `cocopp.htmldesc`
  - [ ] `cocopp.ppconverrorbars`

Feel free to adjust list list.

**Comment by nikohansen** [comment]
Soon there will be `cocopp.data_archive` added ;-)

**Comment by nikohansen** [comment]
The list shows only the first half-or-so, you forget to `<TAB>` again.

**Comment by nikohansen** [comment]
> while `main` should be clearly renamed in the long run

into what?

**Comment by brockho** [comment]
> The list shows only the first half-or-so, you forget to <TAB> again.
Right. I was already overwhelmed such that I thought that's it. The above list is now updated.

**Comment by brockho** [comment]
>> while main should be clearly renamed in the long run

> into what?

What about `postprocess`?

**Comment by nikohansen** [comment]
Out of interest, do we have a usecase for `cocopp.bestalg`? `cocopp.bestalg.<TAB>` or `help(cocopp.bestalg)` or `help(coco.main)` are not particularly helpful to find out.

**Comment by brockho** [comment]
For the future `bestalg` documentation/usecase issue, see also #1604.
